client/src/app/+about/about-peertube/about-peertube.component.scss

```diff
@@ -6,63 +6,14 @@
   margin: auto;
 }
 
-.about-peertube-title {
-  font-size: 25px;
-  text-align: center;
-  font-weight: $font-semibold;
-  margin-bottom: 15px;
-}
-
-.mascot {
-  display: block;
-  margin: 20px auto;
-}
-
-::ng-deep .section-title {
-  font-weight: $font-semibold;
-  font-size: 20px;
-  margin-bottom: 5px;
-}
-
-.description {
-  text-align: center;
-}
-
-.documentation {
-  display: flex;
-  flex-wrap: wrap;
-  margin: 50px 0;
-  justify-content: center;
-
-  .card {
-    margin: 30px;
-    flex-basis: 300px;
-  }
-}
+.card {
+  @include rfs(2rem, margin);
```

Comment:
Is it more appropriate to write `@include margin(2rem, 2rem, 2rem, 2rem)` so that we use the peertube mixin even if the code is repeated infamously ?

Reply:
No your version is better!

You can also update the mixin so it just use `@include rfs($block-start, margin)` if `$inline-end` and other arguments are not set (something like https://stackoverflow.com/a/25402481)